### PR TITLE
feat(flashdb): Added a quickstart for flashdb

### PR DESCRIPTION
**Pull request by mehreentahir16**
feat(flashdb): Added a quickstart for flashdb

# Summary

This is a demo quickstart for the quickstart course.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include an InstallPlan and Documentation reference?
- [x] Are all documentation links publically accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [x] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ ] Did you check that your alerts actually work?

### CODE REVIEW

**Comment by jcountsNR** [comment]
@mehreentahir16 , is this an actual quickstart that we are wanting to merge or is this testing from the docs example only?

File: data-sources/flashdb/config.yml
Comment (aswanson-nr) [inline data-sources/flashdb/config.yml:1]: This should be lowercase
```suggestion
id: flashdb
```

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
Hey @mehreentahir16, I found a few things that need to get cleaned up. If I jumped on this PR to early and it's still in development, please mark it has "draft" until it is ready to be reviewed and merged.

**Comment by mehreentahir16** [comment]
@aswanson-nr @jcountsNR This is just a demo PR for the quickstart course. Please do not merge but I had to go through the review process to make sure I'm not missing steps. I'll fix the PR for you to review it one more time and then update the course based on the process. Thanks

File: quickstarts/flashdb/config.yml
Comment (mehreentahir16) [reply quickstarts/flashdb/config.yml:51]: The only related resources for this quickstart is the course since it's just a demo.

**Comment by aswanson-nr** [comment]
@mehreentahir16 Gotcha! I'm just going to put it in draft to ensure it doesn't get merged, all the workflows and checks will still run as usual!

**Comment by mehreentahir16** [comment]
@aswanson-nr I committed the changes but I still don't see alerts. Can you please check?

File: quickstarts/flashdb/config.yml
Comment (aswanson-nr) [inline quickstarts/flashdb/config.yml:44]: Hey @mehreentahir16! This is why they aren't showing up in the preview!
```suggestion
alertPolicies:
```

**Pull request closed without merging**
